elastalert/ruletypes.py

```diff
@@ -635,10 +635,16 @@ def __init__(self, rule, args=None):
             raise EAException("fields must not be an empty list")
         if type(self.fields) != list:
             self.fields = [self.fields]
-        if self.rules.get('use_terms_query') and (
```

Comment:
Super Nit: I think our preferences differ, but I found the former more slightly more readable, 
same w/r/t doing

```python
if (
    not self.rules.get('query_key').endswith('.keyword') and
    not self.rules.get('query_key').endswith('.raw') and
    self.rules.get('use_keyword_postfix', True)
):
```
below.